src/app/home-page/widget/entry-box/entry-box.component.html

```diff
@@ -43,22 +43,29 @@
     <mat-card-content class="p-3 h-100">
       <div *ngIf="totalEntries !== 0 || isLoading; else noEntries">
         <mat-form-field appearance="outline" class="search-form-field w-100 pb-0">
-          <input
-            matInput
-            data-cy="search-field"
-            [(ngModel)]="filterText"
-            (keyup)="onTextChange($event)"
-            placeholder="Search your {{ entryTypeLowerCase }}s..."
-          />
+          <mat-label>Search your {{ entryTypeLowerCase | titlecase }}s...</mat-label>
+          <input matInput data-cy="search-field" [(ngModel)]="filterText" (keyup)="onTextChange($event)" [matAutocomplete]="auto" />
+          <button *ngIf="filterText" mat-button matSuffix mat-icon-button (click)="clearSearch()">
+            <mat-icon>cancel</mat-icon>
+          </button>
         </mat-form-field>
+        <mat-autocomplete #auto="matAutocomplete">
+          <span>
+            <a *ngFor="let result of listOfResults | slice: 0:5" [routerLink]="allEntriesLink + result.path" class="no-wrap no-underline">
+              <mat-option>{{ result.prettyPath }}</mat-option>
```

Comment:
I wonder if it's worth showing the organization here?

Trying it locally, if I type an organization name, e.g., `coverbeck` or `dockstore`, two GitHub orgs I belong to, I see several matching results in the UI, but I don't see `coverbeck` or `dockstore` displayed. It was a little confusing -- why is it matching something that doesn't look like it matches?

In the screenshot, all the workflows belong to the org `coverbeck`, which is why they are displayed.

![Screen Shot 2023-03-07 at 2 20 56 PM](https://user-images.githubusercontent.com/1049340/223566878-53849765-f319-4986-8f2f-4e6566b4d67b.png)

On the other hand, maybe most users don't belong to multiple repos and typically won't search by org name. And there's a reason the repetitive org name is removed from the mocks :). Still... I don't know, I'll leave it for you to ponder. :)

Reply:
Well, the full path with "github.com" is obviously redundant in this case. I do see the argument for displaying organisation.  In an ideal world, I could see it displaying the organization only when you have conflicts (i.e. if you were searching `md5sum` and you needed the organization to distinguish between `coverbeck/md5sum` and `dockstore/md5sum`. 

Right now, I think that's probably over-engineering things, so I'm inclined to open the ticket (on how to display search results, with the ideas here and above), see if there are user requests, and only improve this on user request.  

The bigger question right now is, if you have multiple workflows with the same name, how does it display in the results below the search box once you press enter?

Reply:
I suppose we could remove the github.com part and just display `coverbeck/md5sum` and `dockstore/md5sum` part for all entries since that's also how we display entries on search:
![image](https://user-images.githubusercontent.com/97123241/223762972-9926c7bb-6375-4013-aaf3-3447262e486d.png)

> The bigger question right now is, if you have multiple workflows with the same name, how does it display in the results below the search box once you press enter?

Not sure I understand this, clicking on an entry in the search results (or pressing enter, wait for changes to be pushed) will navigate you to your workflow or tool page. If they have the same name I'm guessing they would both display in the drop-down but with the same name appearing twice. Like so?
![image](https://user-images.githubusercontent.com/97123241/223764950-842181d6-9221-4e72-8115-97086a38ddf2.png)

Reply:
Agreed, it's fine to wait and see, and yes, I was thinking we could also drop the `github.com`.

My use case is pretty extreme, I suspect I belong to a lot more orgs than the average Dockstore user, and maybe this will be moot most of the time.